force-app/main/default/classes/PlatformEventMonitoringController.cls

```diff
@@ -1,12 +1,21 @@
 /**
 * Custom Controller to show Platform Events Subscriber Status
 **/ 
-public class PlatformEventMonitoringController {
+public with sharing class PlatformEventMonitoringController {
```

Comment:
No downside in "with sharing" as a matter of best practice :-)
In this specific case, EventBusSubscriber effectively has an OWD of Public Read, so it's moot.